core/src/main/scala/cats/Applicative.scala

```diff
@@ -184,11 +172,14 @@ import simulacrum.typeclass
    */
   def whenA[A](cond: Boolean)(f: => F[A]): F[Unit] =
     if (cond) void(f) else pure(())
+
+  override def monoid[A](implicit A: Monoid[A]): Monoid[F[A]] =
```

Comment:
every time we add a method to a trait, we break binary compatibility. Is there any reason to have this here vs the `object`? Can you really customize this based on the `F[_]`?

Reply:
Hmm, maybe you are right. We had the same discussion with regards to parallel, but I think in this case it might not be as customizable.